src/highfive.coffee

```diff
@@ -201,14 +202,15 @@ module.exports = (robot) ->
                     if do_boomerang
                         return tango(robot).order msg, from_obj, from_obj, 10, 'Boomerang'
                         , (order) ->
-                            chatService.boomerang roomid, from_obj, 
+                            chatService.boomerang roomid, from_obj,
                             sheet.logToSheet robot, [
-                                m,
-                                from_obj.email,
-                                from_obj.email,
-                                10,
-                                '(Boomerang)',
-                                order.reward.number,
+                                m,                    # date (why not *new* order date?)
```

Comment:
To answer your question: no good reason. I just didn't think about recording a different time. Having them the same is nice for tracking that the original and the boomerang are related, but a few seconds difference probably wouldn't matter.